docs/components/concepts/job-workers.md

```diff
@@ -77,4 +77,4 @@ The fact that jobs may be worked on more than once means that Zeebe is an "at le
 
 ## Next steps
 
-- [Zeebe overview](./components/zeebe/zeebe-overview.md)
+- [Zeebe overview](components/zeebe/zeebe-overview.md)
```

Comment:
Note that docusaurus doesn't care whether the absolute path starts with `/` or not -- this will look for the linked doc from the root, just as the others that start with `/` do. 

It's accidental that this one doesn't contain a slash, though. I tried to be consistent about specifying `/` throughout the PR. If people would like me to add a `/` here, I would, though I personally don't care enough to change it until requested.

Reply:
Usually, I vote for consistency, but I'm okay with leaving this `/` or no `/` as long as the behavior is consistent. Thanks for calling this out, though! And FYI @christinaausley, this is a good topic to catch during code reviews as we see changes come through but I don't think we need to address this immediately.